server/templates/static/homepage.html

```diff
@@ -57,119 +57,207 @@ <h2>{% trans %}What's new{% endtrans %}</h2>
-          <a class="sub" href="{{ url_for('browser.browser_node', dcid='EncodeBiosampleType') }}">{% trans %}Encyclopedia of DNA Elements (Biosamples){% endtrans %}</a>,
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=PalmerDroughtSeverityIndex_Atmosphere&pc=0&denom=Count_Person&pd=country/USA&pn=United%20States%20of%20America&pt=Country&ept=County">Palmer Drought Severity Index for US Counties</a>
         </li>
-        <li><a class="sub" href="{{ url_for('browser.browser_main') }}">{% trans %}MORE{% endtrans %}</a>
+      </ul>
+    </li>
+    <li>
+      <span class="main">Energy</span>:
+      <ul>
+        <li>
+          <a class="sub" href="{{ url_for('tools.timeline') }}#place=country/IND,country/CHN,country/USA&statsVar=Annual_Generation_Electricity_SolarPhotovoltaic_MainActivityProducer">Energy Generated from Solar in China, India, USA</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#%26svx%3DMean_SolarInsolation%26svy%3DCount_Person_BelowPovertyLevelInThePast12Months%26pcy%3D1%26epd%3DgeoId%2F06%26epn%3DCalifornia%26epts%3DState%26ept%3DCounty%26qd%3D0%26ld%3D0%26dd%3D0%26ct%3D1%26rg%3D0">Poverty vs Solar Energy Potential across California Counties</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=Amount_Production_ElectricityFromOilGasOrCoalSources_AsFractionOf_Amount_Production_Energy&pc=0&denom=Count_Person&pd=Earth&pn=Earth&pt=Place&ept=Country">Energy Produced from Oil, Gas or Coal Worldwide</a>
         </li>
       </ul>
     </li>
-
     <li>
-      {# TRANSLATORS: Header for a set of links to the Data Commons Scatter Plot Explorer tool, where users are able to create scatter plots on many topics. Please see this string in action at https://datacommons.org #}
-      <a class="main" href="{{ url_for('tools.scatter') }}">{% trans project="Data Commons" %}Create Scatter Plots{% endtrans %}</a>:
+      <span class="main">Education</span>:
       <ul>
+        <li>
+          <a class="sub" href="{{ url_for('ranking.ranking', stat_var='Amount_EconomicActivity_ExpenditureActivity_EducationExpenditure_Government_AsFractionOf_Amount_EconomicActivity_ExpenditureActivity_Government', place_type='Country') }}">Countries with Highest Government Spending on Education</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=Count_School&pcx=1&svy=Count_Person_Literate&pcy=1&epd=country/IND&epn=India&epts=Country&ept=AdministrativeArea1&qd=0&ld=0&dd=0&ct=1&rg=0">Literacy vs Schools Per Capita in India</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=DropoutRate_Student_Female_PrimarySchool&svy=Count_School_HasFunctionalToilet&pcy=1&epd=country/IND&epn=India&epts=Country&ept=AdministrativeArea1&qd=0&ld=0&dd=0&ct=0&rg=0">Schools with Functional Toilets vs Female Dropout in India</a>
```

Comment:
I'm slightly puzzled by this... I'd have thought functional toilets vs. total dropout rate ... or functional girls toilets vs. female drop out rate ...